### ISSUE TITLE
Unclear error when running sismic-bdd feature file for a simple cassette player statechart

### ISSUE THREAD

**Issue by tommens** [AlexandreDecan/sismic#81]
When I run 
sismic-bdd player.yaml --steps player_steps.py --features motor_human.feature
(I will give you the corresponding files in a separate mail)
I get an error in one of its scenarios, but the error message is not very clear "CodeEvaluationError", so I do not know how to deal with it. 

 Scenario: Motor turns off when stopped          # motor_human.feature:25
   Given I reproduce "Motor is on while playing" # ../myenv/lib/python3.6/site-packages/sismic/bdd/steps.py:12 0.004s
   When I press stop button                      # ../myenv/lib/python3.6/site-packages/sismic/bdd/wrappers.py:38 0.001s
     Assertion Failed: FAILED SUB-STEP: When I send event stop
     Captured stdout:
     HOOK-ERROR in after_step: CodeEvaluationError: The above exception occurred while executing:
     sent('playReleased')

   Then motor turns off                          # None
   And play button is released                   # None

Failing scenarios:
 motor_human.feature:25  Motor turns off when stopped

**Comment by AlexandreDecan** [comment]
Thanks for reporting this! I'll have a look on why Behave does not properly display the full exception trace.

**Comment by AlexandreDecan** [comment]
Behave has no way to display the full traceback when an exception is built on top of a previous one (as it is the case with our CodeEvaluationError that just encapsulates any exception raised by Python during code evaluation). I saw code in Behave to support "nested exceptions" but this code is currently not used in their ``runner`` implementation. 

As a temporary workaround, I changed the way CodeEvaluationErrors are displayed by default, by adding the message of the underlying exception (ie. should display ``HOOK-ERROR in after_step: CodeEvaluationError: "name 'sent' is not defined" occurred while executing "sent('playReleased')"`` in your case). That's not perfect, but could help.
